tests/test_processing_stack.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+import time
+
+from pytest import mark
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+ENVIRONMENT_TAG_KEY = "EnvironmentType"
+ENVIRONMENT_TAG_VAL = os.environ.get("DEPLOY_ENV", "dev")
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def detect_compute_environment(batch_client):
+    compute_environment_detection_response = batch_client.describe_compute_environments()
+    logger.info("Detected Compute Environments: %s", compute_environment_detection_response)
+
+    for compute_environment in compute_environment_detection_response["computeEnvironments"]:
+        if (
+            compute_environment["tags"][ENVIRONMENT_TAG_KEY] == ENVIRONMENT_TAG_VAL
+            and compute_environment["tags"].get(APPLICATION_TAG_KEY, None) == APPLICATION_TAG_VAL
+        ):
+            datalake_compute_environment = compute_environment
+            logger.info("Datalake Compute Environment: %s", datalake_compute_environment)
+            break
+
+    return datalake_compute_environment
+
+
+def launch_compute_environment_resources(batch_client, datalake_compute_environment):
+    """
+    Trigger immediate launch of computing resources. This will significantly speed up Batch jobs
+    execution.
+    """
+    if (
+        int(datalake_compute_environment["computeResources"]["minvCpus"]) < 1
+        or int(datalake_compute_environment["computeResources"]["desiredvCpus"]) < 1
+    ):
+        batch_client.update_compute_environment(
+            computeEnvironment=datalake_compute_environment["computeEnvironmentName"],
+            computeResources={"minvCpus": 1, "desiredvCpus": 1},
+        )
+
+
+def detect_state_machine(stepfunctions_client):
+    state_machines_detection_response = stepfunctions_client.list_state_machines()
```

Comment:
`assert response["nextToken"] is None` like above.

Reply:
Done in 9f529447d3faf75bb7c0ae62f9b923ae4cbce5c1